src/plugins/auto/executable_network.cpp

```diff
@@ -686,36 +687,90 @@ InferenceEngine::Parameter MultiDeviceExecutableNetwork::GetConfig(const std::st
-                real = _loadContext[CPU].
-                    executableNetwork->GetMetric(name).as<unsigned int>();
                 std::unique_lock<std::mutex> lock(_confMutex);
                 auto deviceInfo =  _loadContext[ACTUALDEVICE].deviceInfo;
                 lock.unlock();
+                bool bthroughput = false;
+                const auto& mode = deviceInfo.config.find(CONFIG_KEY(PERFORMANCE_HINT));
+                if (mode != deviceInfo.config.end() && mode->second == CONFIG_VALUE(THROUGHPUT)) {
+                    bthroughput = true;
+                }
                 if (deviceInfo.deviceName.find("GPU") != std::string::npos) {
                     const auto& mode = deviceInfo.config.find(CONFIG_KEY(PERFORMANCE_HINT));
                     if (mode != deviceInfo.config.end() && mode->second == CONFIG_VALUE(THROUGHPUT)) {
-                         std::map<std::string, InferenceEngine::Parameter> options;
-                         options["MODEL_PTR"] = _network.getFunction(); // CNNntework
-                         try {
-                             auto optimalBatchSize = _core->GetMetric(deviceInfo.deviceName,
-                                     METRIC_KEY(OPTIMAL_BATCH_SIZE), options).as<unsigned int>();
-                             auto rangeOfStreams = _core->GetMetric(deviceInfo.deviceName,
-                                     METRIC_KEY(RANGE_FOR_STREAMS), options).as<std::tuple<unsigned int, unsigned int>>();
-                             real = (std::max)(real, std::get<1>(rangeOfStreams) * optimalBatchSize);
-                         } catch (const InferenceEngine::Exception &iie) {
-                             LOG_WARNING("[AUTOPLUGIN]get optimal infer requset num for GPU auto-batch failed :%s", iie.what());
-                         }
+                        // THROUGHPUT mode
+                        try {
+                            std::map<std::string, InferenceEngine::Parameter> options;
+                            options["MODEL_PTR"] = _network.getFunction();
+                            auto rangeOfStreams = _core->GetMetric(deviceInfo.deviceName,
+                                    METRIC_KEY(RANGE_FOR_STREAMS), options).as<std::tuple<unsigned int, unsigned int>>();
+                            if (std::get<1>(rangeOfStreams) > 0) {
+                                real = 2 * std::get<1>(rangeOfStreams);
+                            } else {
+                                real = defaultNumForTPUT;
+                            }
+                        } catch (const std::exception&) {
+                            LOG_WARNING("[AUTOPLUGIN] GetMetric RANGE_FOR_STREAMS failed");
+                            LOG_WARNING("[AUTOPLUGIN] use default value %u for THROUGHPUT", defaultNumForTPUT);
+                            real = defaultNumForTPUT;
+                        }
+                        // if enable auto-batch, use below code
```

Comment:
no unused code like this, in the main repo, pls

Reply:
This is placeholder for auto-batching enabling. Will update this part in separated PR